chimney/src/test/scala/io/scalaland/chimney/NamingConventionsSpec.scala

```diff
@@ -0,0 +1,34 @@
+package io.scalaland.chimney
+
+import utest._
+import io.scalaland.chimney.dsl._
+
+object NamingConventionsSpec extends TestSuite {
+    
+    val tests = Tests {
```

Comment:
I guess we need larger test coverage, especially:
- multiple name segments, like `foo_bar_baz`
- segments that include numbers
- how this behave in nested transformations
- what about coproducts?